examples/with-emailpassword-vercel/README.md

```diff
@@ -0,0 +1,37 @@
+![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
+
+# SuperTokens EmailPassword Vercel Demo app
+
+This demo app demonstrates deploying a react + nodejs express app on Vercel.
+
+## Project structure & Parameters
+
+-   The frontend code is located in the `src` folder.
+-   The backend API is in the `api/index.js` file .
+-   The `vercel.json` file contains the configurations that allow Vercel to rewrite to push all traffic on `/api` routes to the backend.
+
+## Deploy to Vercel
+
+To deploy this demo application on Vercel, follow the steps outlined below:
+1. Clone or download this project.
+2. Change directory to the **with-emailpassword-vercel** folder.
+     
+
+       cd with-emailpassword-vercel
+          
+
```

Comment:
- Don't you need a step to install all the dependencies and the vercel CLI?
- What about setting up a vercel project on their dashboard and stuff?